specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -379,49 +376,49 @@ directory services and a service provider interface to attach a provider
 of a naming and directory service. The JNDI API is included in Java SE,
 but this specification defines additional requirements.
 
-=== JavaMail™
+=== Jakarta™ Mail
 
 Many Internet applications require the ability
-to send email notifications, so the Java EE platform includes the
-JavaMail API along with a JavaMail service provider that allows an
-application component to send Internet mail. The JavaMail API has two
+to send email notifications, so the Jakarta EE platform includes the
+Jakarta Mail API along with a Jakarta Mail service provider that allows an
+application component to send Internet mail. The Jakarta Mail API has two
 parts: an application-level interface used by the application components
-to send mail, and a service provider interface used at the Java EE SPI
+to send mail, and a service provider interface used at the Jakarta EE SPI
 level.
 
-=== JavaBeans™ Activation Framework (JAF)
+=== Jakarta Activation
```

Comment:
The technology name "JAF" should not be changed.  JAF is not part of Jakarta EE
(nor part of Java EE 8, although it was part of earlier versions).
Since it's still in Java SE 8, this section should just be a reference
to it there.

Reply:
@kwsutter Correct.  JAF, like JAX-WS, and JAXB, and SAAJ, and JWS, is still in Java SE 8.
None of these references should be to the potential Jakarta versions of these specs.

Reply:
Understood. I'm replacing Jakarta Mail with JavaMail on this paragraph too.